### PR TITLE
ControlCenter DataDir Creation when custom

### PR DESCRIPTION
**Pull request by nerdynick**
# Description

Fix issue with Control Center when defining a custom DataDir and the directory needs to be created.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Test Lab Env

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: roles/confluent.control_center/tasks/main.yml [hunk 1]
```diff
@@ -68,6 +68,14 @@
     mode: '764'
     recurse: yes
 
+- name: Create Data Directory
+  file:
+    path: "{{control_center.data_dir_path}}"
```
Comment (domenicbove) [inline roles/confluent.control_center/tasks/main.yml]: Why not just make this task have:
```
path: "{{ control_center['properties']['confluent.controlcenter.data.dir'] }}"
```
and then we do not need a new variable and customers can update that property if they want in the already documented way

is recurse necessary? i am looking to remove it everywhere to improve idempotency. (that can be a separate PR)

**Comment by domenicbove** [comment]
Can we close this one, as it is handled here: https://github.com/confluentinc/cp-ansible/pull/206

**Comment by nerdynick** [comment]
Other solution resolved in #206
